<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryView</class>
 <widget class="QDialog" name="MemoryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>284</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="page">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="followButton">
       <property name="text">
        <string>Follow</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/follow.png</normaloff>:/icons/follow.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">CS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitsView" name="chipSelected" native="true">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showSourcesButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show source</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/code.png</normaloff>:/icons/code.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="autoReloadMode">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Auto Reload off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto Reload on</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auto Reload on and reset</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MemoryPageView" name="memoryPage" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitsView</class>
   <extends>QWidget</extends>
   <header>views/BitsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryPageView</class>
   <extends>QWidget</extends>
   <header>views/MemoryPageView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>loadButton</sender>
   <signal>clicked()</signal>
   <receiver>MemoryView</receiver>
   <slot>onLoadButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>followButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MemoryView</receiver>
   <slot>onFollowButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>page</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MemoryView</receiver>
   <slot>onPageValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showSourcesButton</sender>
   <signal>clicked()</signal>
   <receiver>MemoryView</receiver>
   <slot>onShowSourcesButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoadButtonClicked()</slot>
  <slot>onFollowButtonToggled(bool)</slot>
  <slot>onPageValueChanged(int)</slot>
  <slot>onShowSourcesButtonClicked()</slot>
 </slots>
</ui>
